src/scss/grommet-core/_objects.tabs.scss

```diff
@@ -5,16 +5,33 @@
   padding: 0;
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
   align-items: center;
   list-style: none;
   border-bottom: 1px solid $border-color;
 
   &__content {
-    padding: $inuit-base-spacing-unit;
-
     @include media-query(palm) {
       padding: $inuit-base-spacing-unit 0;
     }
   }
+
+  &--justify-center {
```

Comment:
We typically use SASS a bit more via:

&--justify {
  &-center {
    ...
  }
  &-start {
    ...
  }
...